### ISSUE TITLE
Worldgen Modding: Unpatchable new schematicYoffsets (or am I stupid)

### ISSUE THREAD

**Issue by AreYSerious** [anegostudios/VintageStory-Issues#3561]
### Game Version

v1.19.3

### Platform

Windows

### Modded

Modded

### SP/MP

Singleplayer

### Description

in the structures.json file

1.19 added the new schematicYoffsets which can not be added onto.
just overwritten.

the / in the key don't allow patching into it.

Or thats what im guessing.

Its only possible to completely overwrite the whole thing which doesnt make sense bcs loadorder exists and if multiple mods add stuff into that just one will get loaded.

Im currently just readding all known structures for vanilla and other mods that use it. (BAD)

### How to reproduce

```
[
  {
    "op": "add",
    "path": "/schematicYOffsets",
    "value": {
      "overground/main/cold-climate/coldclimate-o1-stadrian-v1": -1,
      "overground/main/cold-climate/coldclimate-o1-stadrian-v2": -1,
      "overground/main/cold-climate/coldclimate-o1-stadrian-v3": -1,
      "overground/main/cold-climate/coldclimate-o2-stadrian-v4": -2,
      "overground/main/cold-climate/coldclimate-o2-stadrian-v5": -2,
      "overground/main/desert-climate/desertclimate-o1-niclass-small1": -1,
      "overground/main/desert-climate/desertclimate-o1-niclass-small2": -1,
      "overground/main/desert-climate/desertclimate-o1-niclass-small3": -1,
      "overground/main/desert-climate/desertclimate-o1-niclass-small4": -1,
      "overground/main/desert-climate/desertclimate-o1-stadrian-deserthouse1": -1,
      "overground/main/desert-climate/desertclimate-o2-dampus-greek1": -2,
      "overground/main/desert-climate/desertclimate-o2-stadrian-cube1": -2,
      "overground/main/desert-climate/desertclimate-o2-stadrian-deserthouse2": -2,
      "overground/main/desert-climate/desertclimate-o20-stadrian-undergroundwithentrance": -20,
      "overground/main/desert-village/desertvillage-par-o2-0102": -2,
      "overground/main/desert-village/desertvillage-par-o2-0103": -2,
      "overground/main/desert-village/desertvillage-par-o2-0105": -2,
      "overground/main/desert-village/desertvillage-par-o2-0106": -2,
      "overground/main/desert-village/desertvillage-par-o3-0104": -3,
      "overground/main/desert-village/desertvillage-par-o6-0101": -6,
      "overground/main/forest-village/all/forestvillage-par-o1-0105": -1,
      "overground/main/forest-village/all/forestvillage-par-o1-0106": -1,
      "overground/main/forest-village/all/forestvillage-par-o1-0107": -1,
      "overground/main/forest-village/all/forestvillage-par-o1-0303": -1,
      "overground/main/forest-village/all/forestvillage-par-o1-0401": -1,
      "overground/main/forest-village/all/forestvillage-par-o1-0404": -1,
      "overground/main/forest-village/all/forestvillage-par-o1-0604": -1,
      "overground/main/forest-village/all/forestvillage-virrno-o1-0101": -1,
      "overground/main/forest-village/all/forestvillage-virrno-o1-0102": -1,
      "overground/main/forest-village/all/forestvillage-virrno-o1-0103": -1,
      "overground/main/forest-village/all/forestvillage-virrno-o1-0104": -1,
      "overground/main/forest-village/all/forestvillage-virrno-o1-0201": -1,
      "overground/main/forest-village/all/forestvillage-virrno-o1-0202": -1,
      "overground/main/forest-village/all/forestvillage-virrno-o1-0203": -1,
      "overground/main/forest-village/all/forestvillage-virrno-o1-0204": -1,
      "overground/main/forest-village/all/forestvillage-virrno-o1-0205": -1,
      "overground/main/forest-village/all/forestvillage-virrno-o1-0403": -1,
      "overground/main/forest-village/all/forestvillage-virrno-o1-0501": -1,
      "overground/main/forest-village/all/forestvillage-virrno-o10-0402": -10,
      "overground/main/forest-village/all/forestvillage-virrno-o2-0301": -2,
      "overground/main/forest-village/all/forestvillage-virrno-o2-0302": -2,
      "overground/main/forest-village/all/forestvillage-virrno-o5-0503": -5,
      "overground/main/forest-village/all/forestvillage-virrno-o5-0603": -5,
      "overground/main/forest-village/all/forestvillage-virrno-o6-0602": -6,
      "overground/main/forest-village/all/forestvillage-virrno-o8-0502": -8,
      "overground/main/forest-village/all/forestvillage-virrno-o8-0601": -8,
      "overground/main/forest-village/windmills/forestvillage-virrno-o2-0701": -2,
      "overground/main/forest-village/windmills/forestvillage-virrno-o2-0702": -2,
      "overground/main/forest-village/windmills/forestvillage-virrno-o2-0703": -2,
      "overground/main/largeruins/largeruins-dampus-o3-0403": -3,
      "overground/main/largeruins/largeruins-dampus-o4-0103": -4,
      "overground/main/largeruins/largeruins-dampus-o4-0203": -4,
      "overground/main/largeruins/largeruins-niclass-o3-0101": -3,
      "overground/main/largeruins/largeruins-niclass-o3-0301": -3,
      "overground/main/largeruins/largeruins-niclass-o3-0401": -3,
      "overground/main/largeruins/largeruins-niclass-o3-0501": -3,
      "overground/main/largeruins/largeruins-niclass-o3-0503": -3,
      "overground/main/largeruins/largeruins-niclass-o3-0601": -3,
      "overground/main/largeruins/largeruins-niclass-o3-0603": -3,
      "overground/main/largeruins/largeruins-niclass-o3-0701": -3,
      "overground/main/largeruins/largeruins-niclass-o3-0703": -3,
      "overground/main/largeruins/largeruins-quentin-o1-0201": -1,
      "overground/main/largeruins/largeruins-samgrey-o3-0102": -3,
      "overground/main/largeruins/largeruins-samgrey-o3-0202": -3,
      "overground/main/largeruins/largeruins-samgrey-o3-0302": -3,
      "overground/main/largeruins/largeruins-samgrey-o3-0402": -3,
      "overground/main/largeruins/largeruins-samgrey-o3-0502": -3,
      "overground/main/largeruins/largeruins-vil-o3-0602": -3,
      "overground/main/largeruins/largeruins-vil-o3-0702": -3,
      "overground/main/largeruins/largeruins-vil-o4-0303": -4,
      "overground/main/mediumruins/mediumruins-bell-o1-0304": -1,
      "overground/main/mediumruins/mediumruins-dampus-o1-0804": -1,
      "overground/main/mediumruins/mediumruins-dampus-o1-0904": -1,
      "overground/main/mediumruins/mediumruins-dampus-o2-0503": -2,
      "overground/main/mediumruins/mediumruins-dampus-o2-0603": -2,
      "overground/main/mediumruins/mediumruins-dampus-o2-0704": -2,
      "overground/main/mediumruins/mediumruins-niclass-o1-0104": -1,
      "overground/main/mediumruins/mediumruins-niclass-o1-0204": -1,
      "overground/main/mediumruins/mediumruins-niclass-o1-0304": -1,
      "overground/main/mediumruins/mediumruins-niclass-o1-0404": -1,
      "overground/main/mediumruins/mediumruins-niclass-o1-0504": -1,
      "overground/main/mediumruins/mediumruins-niclass-o1-0604": -1,
      "overground/main/mediumruins/mediumruins-niclass-o2-0101": -2,
      "overground/main/mediumruins/mediumruins-niclass-o2-0102": -2,
      "overground/main/mediumruins/mediumruins-niclass-o2-0202": -2,
      "overground/main/mediumruins/mediumruins-niclass-o2-0301": -2,
      "overground/main/mediumruins/mediumruins-niclass-o2-0302": -2,
      "overground/main/mediumruins/mediumruins-niclass-o2-0303": -2,
      "overground/main/mediumruins/mediumruins-niclass-o2-0401": -2,
      "overground/main/mediumruins/mediumruins-niclass-o2-0402": -2,
      "overground/main/mediumruins/mediumruins-niclass-o2-0403": -2,
      "overground/main/mediumruins/mediumruins-niclass-o2-0501": -2,
      "overground/main/mediumruins/mediumruins-niclass-o2-0502": -2,
      "overground/main/mediumruins/mediumruins-niclass-o2-0601": -2,
      "overground/main/mediumruins/mediumruins-niclass-o2-0602": -2,
      "overground/main/mediumruins/mediumruins-niclass-o2-0701": -2,
      "overground/main/mediumruins/mediumruins-niclass-o2-0801": -2,
      "overground/main/mediumruins/mediumruins-niclass-o2-0802": -2,
      "overground/main/mediumruins/mediumruins-niclass-o2-0901": -2,
      "overground/main/mediumruins/mediumruins-niclass-o2-0902": -2,
      "overground/main/mediumruins/mediumruins-niclass-o3-0201": -3,
      "overground/main/mediumruins/mediumruins-niclass-o3-0203": -3,
      "overground/main/mediumruins/mediumruins-niclass-o3-0204": -3,
      "overground/main/mediumruins/mediumruins-niclass-o3-0702": -3,
      "overground/main/mediumruins/mediumruins-niclass-o4-0104": -4,
      "overground/main/mediumruins/mediumruins-vil-o2-0103": -2,
      "overground/main/mediumruins/mediumruins-vil-o3-0703": -3,
      "overground/main/mediumruins/mediumruins-vil-o3-0803": -3,
      "overground/main/mediumruins/mediumruins-vil-o3-0903": -3,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o1-0302": -1,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o1-0304": -1,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o1-0306": -1,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o11-0503": -11,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o12-0502": -12,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o2-0401": -2,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o2-0402": -2,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o22-0504": -22,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o3-0101": -3,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o3-0102": -3,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o3-0103": -3,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o3-0404": -3,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o4-0203": -4,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o4-0405": -4,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o4-0501": -4,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o4-0505": -4,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o5-0204": -5,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o6-0104": -6,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o6-0105": -6,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o6-0202": -6,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o6-0205": -6,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o6-0301": -6,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o6-0303": -6,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o6-0305": -6,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o6-0403": -6,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o6-0406": -6,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o7-0206": -7,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o8-0106": -8,
      "overground/main/oldvanilla-village/villagesvanilla-niclass-o8-0201": -8,
      "overground/main/road-for-village/road-niclass-o1-0101": -1,
      "overground/main/road-for-village/road-niclass-o1-0102": -1,
      "overground/main/road-for-village/road-niclass-o1-0103": -1,
      "overground/main/road-for-village/road-niclass-o1-0104": -1,
      "overground/main/road-for-village/road-niclass-o1-0105": -1,
      "overground/main/road-for-village/road-niclass-o1-0106": -1,
      "overground/main/road-for-village/road-niclass-o1-0107": -1,
      "overground/main/translocators/o1/translocator-niclass-o1-0101": -1,
      "overground/main/translocators/o1/translocator-niclass-o1-0102": -1,
      "overground/main/translocators/o1/translocator-niclass-o1-0103": -1,
      "overground/main/translocators/o1/translocator-niclass-o1-0104": -1,
      "overground/main/translocators/o2/translocator-niclass-o2-0201": -2,
      "overground/main/translocators/o2/translocator-niclass-o2-0202": -2,
      "overground/main/translocators/o2/translocator-niclass-o2-0203": -2,
      "overground/main/translocators/o2/translocator-niclass-o2-0204": -2,
      "overground/main/translocators/o2/translocator-niclass-o2-0205": -2,
      "overground/main/translocators/o2/translocator-niclass-o2-0206": -2,
      "overground/main/translocators/o2/translocator-niclass-o2-0207": -2,
      "overground/main/translocators/o2/translocator-niclass-o2-0208": -2,
      "overground/main/translocators/o2/translocator-niclass-o2-0209": -2,
      "overground/main/translocators/o2/translocator-niclass-o2-0210": -2,
      "overground/main/translocators/o2/translocator-niclass-o2-0211": -2,
      "overground/main/translocators/o2/translocator-niclass-o2-0212": -2,
      "overground/main/translocators/o4/translocator-niclass-o4-0301": -4,
      "overground/main/translocators/o4/translocator-niclass-o4-0302": -4,
      "overground/main/vsvillage-village/houses/vsvillage-niclass-o1-0101": -1,
      "overground/main/vsvillage-village/houses/vsvillage-niclass-o1-0102": -1,
      "overground/main/vsvillage-village/houses/vsvillage-niclass-o2-0201": -2,
      "overground/main/vsvillage-village/houses/vsvillage-niclass-o2-0202": -2,
      "overground/main/vsvillage-village/houses/vsvillage-niclass-o2-0203": -2,
      "overground/main/vsvillage-village/houses/vsvillage-niclass-o2-0204": -2,
      "overground/main/vsvillage-village/houses/vsvillage-niclass-o2-0205": -2,
      "overground/main/vsvillage-village/houses/vsvillage-niclass-o2-0206": -2,
      "overground/main/vsvillage-village/houses/vsvillage-niclass-o2-0301": -2,
      "overground/main/vsvillage-village/houses/vsvillage-niclass-o2-0302": -2,
      "overground/main/vsvillage-village/houses/vsvillage-niclass-o2-0303": -2,
      "overground/main/vsvillage-village/houses/vsvillage-niclass-o2-0304": -2,
      "overground/main/vsvillage-village/houses/vsvillage-niclass-o2-0305": -2,
      "overground/main/vsvillage-village/mainbuildings/vsvillage-niclass-o3-0501": -3,
      "overground/main/vsvillage-village/wallsegments/vsvillage-niclass-o1-0401": -1,
      "overground/main/vsvillage-village/wallsegments/vsvillage-niclass-o1-0402": -1,
      "overground/main/vsvillage-village/wells/vsvillage-niclass-o10-0601": -10,

















      "surface/agricultural/common/animal-pen1": -2,
      "surface/agricultural/common/animal-pen2": -7,
      "surface/agricultural/common/elevated-farmland1": -6,
      "surface/agricultural/common/elevated-farmland2": -2,
      "surface/agricultural/common/elevated-farmland3": -2,
      "surface/agricultural/common/elevated-farmland4": -2,
      "surface/agricultural/common/farmer-shed1": -2,
      "surface/agricultural/common/farmer-shed2": -6,
      "surface/agricultural/common/farmer-shed3": -18,
      "surface/agricultural/common/silo1": -1,
      "surface/agricultural/uncommon/windmill1": -2,
      "surface/agriculturallarge/elevated-farmland-large1": -2,
      "surface/ancient/descent": -4,
      "surface/ancient/eight-gods": -2,
      "surface/ancient/godrays-gate": -2,
      "surface/ancient/sacrifice-mound": -2,
      "surface/ancient/stonehenge": -2,
      "surface/ancient/tear-drop": -18,
      "surface/ancient/the-aging-pathway": -2,
      "surface/aqueduct/aqueduct-section1": -2,
      "surface/aqueduct/aqueduct-section2": -2,
      "surface/aqueduct/aqueduct-section3": -2,
      "surface/aqueduct/bridge-section1": -2,
      "surface/aqueduct/bridge-section2": -2,
      "surface/aqueduct/bridge-section3": -2,
      "surface/civic/common/center-building1": -2,
      "surface/civic/common/center-building2": -4,
      "surface/civic/common/center-building3": -3,
      "surface/civic/common/center-building4": -5,
      "surface/civic/uncommon/factory1": -6,
      "surface/civic/uncommon/factory2": -7,
      "surface/civic/uncommon/factory3": -5,
      "surface/civic/rare/fountain1": -8,
      "surface/civic/common/graveyard1": -8,
      "surface/civic/common/graveyard2": 0,
      "surface/civic/uncommon/monument1": -2,
      "surface/civic/veryrare/protected-well1": -5,
      "surface/civic/uncommon/storage1": -5,
      "surface/civic/common/supply-room1": -8,
      "surface/civic/rare/tawern1": -6,
      "surface/civic/veryrare/theatre1": -2,
      "surface/housemedium/medium1": -5,
      "surface/housemedium/medium2": -2,
      "surface/housemedium/medium3": -7,
      "surface/housemedium/medium4": -6,
      "surface/housemedium/medium5": -4,
      "surface/housemedium/medium6": -7,
      "surface/housemedium/medium7": -3,
      "surface/military/uncommon/barbican1": -7,
      "surface/military/uncommon/barbican2": -7,
      "surface/military/uncommon/keep1": -6,
      "surface/military/rare/keep2": -5,
      "surface/military/common/tower1": -5,
      "surface/military/common/tower2": -2,
      "surface/military/common/wall-section1": -5,
      "surface/military/common/wall-section2": -2,
      "surface/military/common/wall-section3": -2,
      "surface/military/common/wall-section4": -2,
      "surface/military/common/wall-section5": -2,
      "surface/military/common/wall-section6": -2,
      "surface/religious/common/common1": -5,
      "surface/religious/common/common2": -2,
      "surface/religious/rare/holy-burial1": -4,
      "surface/religious/rare/holy-burial2": -3,
      "surface/religious/rare/holy-healer1": -6,
      "surface/religious/common/monument1": -1,
      "surface/religious/common/monument2": -2,
      "surface/religious/common/sacrifical-place1": -5,
      "surface/religious/uncommon/unknown1": -6,
      "surface/religious/uncommon/unknown2": -5,
      "surface/religiouslarge/veryrare/chonky1": -2,
      "surface/religiouslarge/veryrare/chonky2": -8,
      "surface/religiouslarge/veryrare/chonky3": -6,
      "surface/religiouslarge/veryrare/chonky4": -7,
      "surface/religiouslarge/veryrare/chonky5": -6,
      "surface/religiouslarge/legendary/unknown-chonky1": -13,
      "surface/housesmall/small1": -5,
      "surface/housesmall/small2": -2,
      "surface/housesmall/small3": -2,
      "surface/housesmall/small4": -2,
      "surface/housesmall/small5": -2,
      "surface/housesmall/small6": -2,
      "surface/housesmall/small7": -2,
      "surface/housesmall/small8": -2,
      "surface/housesmall/small9": -7,
      "surface/housesmall/small10": -2,
      "surface/housesmall/small11": -2,
      "surface/housesmall/small12": -6,
      "surface/housesmall/small13": -7,
      "surface/housesmall/small14": -5,
      "surface/housesmall/small15": -2,
      "surface/standalone/alien-portal1": -2,
      "surface/standalone/monument-s1": -2,
      "surface/standalone/monument-s2": -2,
      "surface/standalone/sacrifical-place-s1": -7,
      "surface/standalone/witch-hut1": -9,
      "surface/housetiny/tiny1": -1,
      "surface/housetiny/tiny2": -1,
      "surface/housetiny/tiny3": -1,
      "surface/housetiny/tiny4": -1,
      "surface/housetiny/tiny5": -1,
      "surface/housetiny/tiny6": -1,
      "surface/housetiny/tiny7": -1,
      "surface/housetiny/tiny8": -1,
      "surface/housetiny/tiny9": -1,
      "surface/housetiny/tiny10": -2,
      "surface/housetiny/tiny11": -1,
      "surface/housetiny/tiny12": -1,
      "surface/housetiny/tiny13": -1,
      "surface/housetiny/tiny14": -1,
      "surface/housetiny/tiny15": -1,
      "surface/housetiny/tiny16": -1,
      "surface/housetiny/tiny17": -1,
      "surface/housetiny/tiny18": -1,
      "surface/housetiny/tiny19": -1,
      "surface/housetiny/tiny20": -1,
      "surface/housetiny/tiny21": -1,
      "surface/housetiny/tiny22": -1,
      "surface/housetiny/tiny23": -1,
      "surface/housetiny/tiny24": -1,
      "surface/housetiny/tiny25": -1,
      "surface/housetiny/tiny26": -2,
      "surface/housetiny/tiny27": -1,
      "surface/housetiny/tiny28": -1,
      "surface/housetiny/tiny29": -1,
      "surface/housetiny/tiny30": -1,
      "surface/housetiny/tiny31": -1,
      "surface/housetiny/tiny32": -2,
      "surface/housetiny/tiny33": -1,
      "surface/housetiny/tiny34": -1,
      "surface/housetiny/tiny35": -1,
      "surface/housetiny/tiny36": -2,
      "surface/housetiny/tiny37": -1,
      "surface/housetiny/tiny38": -3,
      "surface/housetiny/tiny39": -1,
      "surface/housetiny/tiny40": -1,










      "overground/andesite1": 0,
			"overground/andesite2": -4,
			"overground/basalt1": -1,
			"overground/basalt2": -1,
			"overground/bauxite1": -1,
			"overground/bauxite2": -1,
			"overground/chalk1": -1,
			"overground/chalk2": -4,
			"overground/chert1": -5,
			"overground/chert2": -1,
			"overground/claystone1": -1,
			"overground/claystone2": -2,
			"overground/conglomerate1": -6,
			"overground/conglomerate2": -4,
			"overground/granite1": -1,
			"overground/granite2": -1,
			"overground/kimberlite1": 0,
			"overground/kimberlite2": -2,
			"overground/limestone1": -1,
			"overground/limestone2": -1,
			"overground/marble1": -1,
			"overground/peridotite1": -4,
			"overground/peridotite2": -1,
			"overground/phyllite1": -1,
			"overground/phyllite2": -6,
			"overground/sandstone1": -3,
			"overground/sandstone2": -1,
			"overground/shale1": -4,
			"overground/shale2": -1,
			"overground/slate1": -2,
			"overground/slate2": 0,
			"overground/suevite1": -4,
			"overground/suevite2": -1,
			"overground/broken-trader-cart1": 0,
			"overground/bear1": -1,
			"overground/wolf1": -2,
			"overground/vixic": -5,
			"overground/burnttower": -3,
			"overground/sandpeat": -5,
			"overground/skulltower": -2,
			"overground/laboratory": -1,
			"overground/zengarden": -4,
			"overground/windmill": -1,
			"overground/toxic-laboratory": -5,
			
			"caravans/caravan1": -1,
			"caravans/caravan2": 0,
			"caravans/caravan3": -1,
			"caravans/caravan4": -1,
			"caravans/caravan5": -1,
			"caravans/caravan6": 0,
			"caravans/caravan7": -1,
			"caravans/caravan8": 0,
			"caravans/caravan9": -1,
			"caravans/caravan10": 0,
			"caravans/caravan11": 0,
			"caravans/caravanclutter1": -1,
			"caravans/caravanclutter2": -1,
			"caravans/caravanclutter3": -4,
			"caravans/caravanclutter4": -5,
			"caravans/caravanclutter5": -1,
			"caravans/caravanclutter6": -1,
			"caravans/caravanclutter7": -1,
			"caravans/caravanclutter8": -1,
			"caravans/caravanclutter9": -1,
			
			"battleground/battlefield-locusts": -1,
			"battleground/battlefield-trunk": -1,
			"battleground/battleground-banner-fish": -2,
			"battleground/battleground-banner-keys": -1,
			"battleground/battleground-supplies": -1,
			"battleground/brass-staff": 0,
			"battleground/copper-shield": 0,
			"battleground/dead-eidolon": -1,
			"battleground/dirt-cube": -3,
			"battleground/giant-belt": 0,
			"battleground/obsidian-sword": -3,
			"overground/last-stand": -1
    },
    "file": "game:worldgen/structures.json"
  }
]
```

### Screenshots

![image](https://github.com/anegostudios/VintageStory-Issues/assets/100879715/b66456f9-b82c-4446-9e21-213e368cddea)

### Logs

```Log```

**Comment by AreYSerious** [comment]
Will be fixed in 1.20
Ty

**Comment by tyronx** [comment]
yea, thanks for the report, I believe this should be fixed in the upcoming 1.20, by using op `addmerge`. Otherwise do reopen once 1.20 is out.
